### PR TITLE
use query-don + usage in the browser

### PR DESCRIPTION
**Pull request by kesla**
Use query-dom for parsing instead of a homegrown DOM.

R = @iefserge
T = patch

### CODE REVIEW

**Comment by iefserge** [comment]
@kesla why are the node names capitalized? what's the advantage of this? I'd argue this makes it harder to read, and easier to mistype 'a' instead of 'A'.

**Comment by kesla** [comment]
@iefserge it's to be compatible with the browser dom. Might not be needed for the `getElementsByTagName` stuff - but for `.tagName` I think it makes sense to be compatible with the browser dom.

**Comment by iefserge** [comment]
@kesla I just tried in Firefox `document.querySelector('html').tagName` and got `html` as a result. Safari and Chrome return `HTML`.

**Comment by kesla** [comment]
@iefserge hey, thanks a lot for looking up. That's super weird. I'll update this with something nice.

File: lib/embeds/types/custom.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import url from 'url';
 import getDimensions from '../dimensions';
 
 export default elm => {
-  if (elm.nodeName !== 'iframe') {
+  if (elm.tagName.toLowerCase() !== 'iframe') {
```
Comment (kesla) [inline lib/embeds/types/custom.js:5]: changed to `tagName` for consistency

**Comment by kesla** [comment]
@iefserge RFR

**Comment by kesla** [comment]
@iefserge hey! Would love it if you could review this :)

Not only is this using query-don now - it's also compatible with the browser! :dancers:

**Comment by iefserge** [comment]
@kesla  looks great!

**Comment by kesla** [comment]
Thank you sir!

**Pull request merged**